sql/core/src/test/resources/sql-tests/inputs/subquery/in-subquery/in-group-by.sql

```diff
@@ -0,0 +1,117 @@
+-- A test suite for GROUP BY in parent side, subquery, and both predicate subquery
+-- It includes correlated cases.
+
+-- tables and data types
+
+CREATE DATABASE indb;
```

Comment:
I guess what's missing here is the `USE indb` after the `CREATE DATABASE` to direct all the objects created after this statement to have a different database/schema. My bad. I thought `CREATE DATABASE` implicitly switch to a new database/schema.